src/Resources/skeleton/doctrine/Repository.tpl.php

```diff
@@ -29,10 +29,10 @@ public function __construct(ManagerRegistry $registry)
      * @throws ORMException
      * @throws OptimisticLockException
      */
-    public function add(<?= $entity_class_name ?> $entity, bool $flush = true): void
+    public function add(<?= $entity_class_name ?> $entity, bool $flush = false): void
     {
         $this->_em->persist($entity);
-        if ($flush) {
+        if (!$flush) {
```

Comment:
```
if (no flush requested) then
    flush transaction
```
Something is wrong there?

Reply:
doing things at 2AM is not a good idea 🤦